src/app/views/home/Home.js

```diff
@@ -120,6 +120,21 @@ export default class Home extends React.Component {
       </a>
     )
   }
+  /**
+   * 
+   * @param {*} URL url of respective application
+   * @param {*} text apply as {text}
+   * @returns button of respective application
+   */
+
+  ApplyButton(URL, text) {
+    return (
+      <Button className="apply-btn"
+        href={URL} target="_blank" rel="noopener noreferrer">
+          {text}
+      </Button>
+      )
+  }
```

Comment:
This should be a function in a separate file that gets exported. Put it in the `/components` folder.